### PR TITLE
Clipboard — bindings

### PR DESCRIPTION
**Pull request by mobily**
This PR adds `Clipboard` bindings.

API:
https://facebook.github.io/react-native/docs/clipboard.html

Usage examples:
```
Clipboard.setString("ReasonML");
Clipboard.getString()
|> Js.Promise.then_(text => {
   Js.log(text);
   Js.Promise.resolve();
 });
```

### CODE REVIEW

File: src/clipboard.re [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[@bs.module "react-native"] [@bs.scope "Clipboard"]
+external getString : unit => Js.Promise.t(string) = "";
```
Comment (wokalski) [inline src/clipboard.re]: Does it ever throw?

File: src/clipboard.rei [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[@bs.module "react-native"] [@bs.scope "Clipboard"]
```
Comment (wokalski) [inline src/clipboard.rei]: Our convention is to use lets in the `rei`s. `external` is just an implementation detail

File: src/clipboard.re (hunk 1 above)
Comment (mobily) [reply src/clipboard.re]: it doesn't, and it returns an empty string (in Promise) if there's nothing in the clipboard

**Comment by mobily** [comment]
@wokalski ✔️ fixed!

**Comment by mobily** [comment]
thanks!

**Pull request merged**